src/style/style.css

```
@@ -15,9 +16,76 @@ html, body {
     display: flex;
     flex-direction: column;
     justify-content: center;
+    align-items: center;
+}
+
+.followers-num {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 23px;
+}
+
+.followings-num {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 23px;
 }
```

```suggestion
.followers-num, .followings-num {
    font-style: normal;
    font-weight: bold;
    font-size: 18px;
    line-height: 23px;
}
```

적용하고자 하는 스타일이 똑같으니까 하나로 합쳐도 될거 같습니다